src/app/camera.cpp

```diff
@@ -69,6 +112,10 @@ glm::vec3 Camera::getScreenCenter() const {
     return screen_center_m;
 }
 
+glm::vec3 Camera::getXAxis() const {
+    return glm::normalize(glm::cross(direction_m, screen_up_m));
+}
+
 void Camera::updateCameraGeometry() {
     screen_u_m = glm::normalize(glm::cross(direction_m, screen_up_m));
     screen_v_m = glm::normalize(glm::cross(direction_m, screen_u_m));
```

Comment:
Same redundant glm::normalize
```suggestion
    screen_u_m = glm::cross(direction_m, screen_up_m);
    screen_v_m = glm::cross(direction_m, screen_u_m);
```

Reply:
Fixed in 4163279